packages/datadog_flutter_plugin/lib/src/rum/rum_user_action_detector.dart

```diff
@@ -405,15 +412,25 @@ Element? _findGestureDetectorElement(
 class RumUserActionAnnotation extends StatelessWidget {
   final String description;
   final Widget child;
+  final Map<String, Object?>? attributes;
```

Comment:
One more comment -- you did such a good job with the doc strings on your other PR it made me realize we should add some documentation for this as well. You should be good to just add it to the class's doc string.